README.md

```diff
@@ -1,6 +1,58 @@
-# tds-community
+# TELUS Design System Community
 
-TDS community components
+The TELUS Design System (TDS) is a set of living guidelines that communicates our brand promise through our digital experiences.
+It's a holistic platform that integrates documentation, guidelines and design management that serves as single source for
+digital design guidelines, code patterns and UI elements.
+
+## Introduction
+
+The TDS Community is a collaborative platform where TELUS designers and developers can rapidly discover, share, and consume
+their modules as part of the TELUS Design System. This includes React components, the [component library for designers](https://tds.telus.com/getting-started/designers.html), and their documentation.
+
+TDS Community is similar to [TDS Core](https://github.com/telusdigital/tds-core) since community components
+have the same technical and branding foundations as core components, however the underlying governance model and
+maintenance workflow differs. Here is a high level summary:
+
+| Feature               | TDS Core      | TDS Community                |
+| --------------------- | ------------- | ---------------------------- |
+| Component Reusability | High          | Moderate                     |
+| Release frequency     | Low           | High                         |
```

Comment:
I suggest saying "Rate of change" instead of "Release frequency". I don't think we want to give the impression that we won't be making core releases anymore, but instead the message is that there is more instability in community components.